integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -178,6 +178,7 @@ case class CarbonLoadDataCommand(
         // First system has to partition the data first and then call the load data
         LOGGER.info(s"Initiating Direct Load for the Table : ($dbName.$tableName)")
         GlobalDictionaryUtil.updateTableMetadataFunc = updateTableMetadata
+        DataLoadingUtil.deleteLoadsAndUpdateMetadata(isForceDeletion = false, table)
```

Comment:
Please add the purpose why we have to move deletion call from CarbonDataRDDFactory.scala to here